packit/source_git.py

```diff
@@ -74,9 +74,12 @@ def __init__(
         self.source_git = source_git
         self.dist_git = dist_git
         self.upstream_ref = upstream_ref
-        self.upstream_url = upstream_url or next(
-            self.source_git.remote(upstream_remote or "origin").urls
-        )
+        try:
+            self.upstream_url = upstream_url or next(
+                self.source_git.remote(upstream_remote or "origin").urls
+            )
+        except ValueError as exc:
+            raise PackitException(exc)
```

Comment:
I'd appreciate an error message for the user which would describe the situation, something like

> Provided source-git repo does not seem to be a clone of the upstream repository because there are no git remotes in the repository. Please clone the upstream repository or add the origin remote.

Reply:
I've used your proposed error description. Thank you for the error text.